src/service_aggregator.py

```diff
@@ -174,7 +173,7 @@ async def entry(message, guid, coalesce_type, caller) -> (dict, int):
     # We told the world what we can do!
     # Workflow will be a list of the functions, and the parameters if there are any
 
-    read_from_cache = bypass_cache or overwrite_cache
+    read_from_cache = not (bypass_cache or overwrite_cache)
```

Comment:
So now the variable name makes sense, but the logic below is wrong. If `read_from_cache` is true, then we want to get cached results.

Reply:
Oof, I'm really swimming upstream on this one.